distribution/lib/Standard/Test/0.0.0-dev/src/Bench.enso

```diff
@@ -30,8 +32,12 @@ measure = ~act -> label -> iter_size -> num_iters ->
         res = iter_size.times single_call
         avg = avg_list res
         fmt = (avg / 1000000).format "%.2f"
+        result.put (result.get + avg)
         IO.println (label + "/iteration:" + act_it_num.to_text + ": " + fmt + "ms")
     num_iters.times iteration
```

Comment:
I'm wondering if it wouldn't be simpler to change `num_iters.times iteration` to `0.up_to num_iters . map iteration` and then return the `avg` as a result of each iteration - getting a Vector of iteration times. Then you could just use `vec.compute Mean`.

But that is just a suggestion to user a bit 'higher-level' concepts to make the code more direct about what it is computing, current implementation looks ok too.